lib/engine/game/g_1825/game.rb

```diff
@@ -1176,6 +1176,11 @@ def check_bankrupt!(entity)
           entity.close!
         end
 
+        def hex_blocked_by_ability?(_entity, abilities, hex)
+          abilities = Array(abilities)
+          abilities.any? { |ability| ability.hexes.include?(hex.id) }
```

Comment:
can you make this a one liner Array(...)any same with the one below

Reply:
Yep